TripKit/Classes/helpers/TKUserProfileHelper.swift

```diff
@@ -70,16 +76,39 @@ public class TKUserProfileHelper: NSObject {
   
   @objc public class func setModeIdentifier(_ modeIdentifier: Identifier, toHidden hidden: Bool) {
     update(hiddenModeIdentifiers, forKey: .hidden, modeIdentifier: modeIdentifier, include: hidden)
```

Comment:
Please add a unit test for this logic to make sure it’s working as expected and doesn’t break down the road.